client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+    <div>
+        <h4>{{ l("Edit Collection Attributes") }}</h4>
+        <b-alert show variant="warning" dismissible>
+            {{
+                l(
+                    "Collections are immutable. This means there will be some things you cannot change without creating a new collection. "
+                )
+            }}
+        </b-alert>
+        <div v-if="jobError">
+            <b-alert show variant="danger" dismissible>
+                {{ errorMessage }}
+            </b-alert>
+        </div>
+        <b-tabs content-class="mt-3">
+            <b-tab>
+                <template v-slot:title> <i class="fa fa-bars"></i>{{ l(" Attributes") }}</template>
+                <b>{{ l("Name: ") }}</b> <i>{{ collectionName }}</i>
+                <br />
+                <b>{{ l("Collection Type: ") }}</b> <i>{{ collectionType }}</i>
+                <br />
+                <b>{{ l("Elements: ") }}</b> <br />
+                <div v-for="element in collectionElements" :key="element">{{ element.element_identifier }} <br /></div>
+            </b-tab>
+            <b-tab>
+                <template v-slot:title> <i class="fa fa-table"></i>{{ l(" Database/Build") }}</template>
+                <div class="alert alert-secondary" role="alert">
+                    <div class="float-left">Change database/genome of all elements in collection</div>
```

Comment:
I think we want to be consistent here in uppercase/lowercase and I think we usually call it Database/Build ?
upper/lowecase is not my strongsuit, but it should probably most be uppercase for Database, Genome and Collection ?